### PR TITLE
Add BeeWi SmartClim BLE sensor documentation

### PR DESCRIPTION
**Pull request by alemuro**
**Description:**

This PR adds the documentation for the new BeeWi SmartClim BLE sensor. This documentation is based on the one for `mitemp_bt`.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26174

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10222"></a>

### CODE REVIEW

File: source/_components/beewi_smartclim.markdown [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: "BeeWi SmartClim BLE"
+description: "Instructions on how to integrate MBeeWi SmartClim BLE sensor with Home Assistant."
+logo: beewi_by_otio.png
+ha_category:
+  - Sensor
+ha_release: 0.98
```
Comment (klaasnicolaas) [inline source/_components/beewi_smartclim.markdown]:
```suggestion
ha_release: 0.99
```

File: source/_components/beewi_smartclim.markdown [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/beewi_smartclim.markdown]: Please remove the `redirect_from`

File: source/_components/beewi_smartclim.markdown [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+Check for `BeeWi SmartClim` or similar entries, those are your sensor.
+
+To use your Mi Temperature and Humidity sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: beewi_smartclim
+    mac: 'xx:xx:xx:xx:xx:xx'
+    monitored_conditions:
```
Comment (klaasnicolaas) [inline source/_components/beewi_smartclim.markdown]: Please remove all optional config variables from the first example

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/beewi_smartclim.markdown (hunk 1 above)
Comment (alemuro) [reply source/_components/beewi_smartclim.markdown]: Done 😄

File: source/_components/beewi_smartclim.markdown (hunk 2 above)
Comment (alemuro) [reply source/_components/beewi_smartclim.markdown]: Done 😄

File: source/_components/beewi_smartclim.markdown (hunk 3 above)
Comment (alemuro) [reply source/_components/beewi_smartclim.markdown]: Done 😄

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @alemuro! Looks great!

**Pull request merged**